Doc/c-api/unicode.rst

```diff
@@ -627,7 +627,7 @@ APIs:
    .. versionadded:: 3.3
 
 
-.. c:function:: PyObject* PyUnicode_Substring(PyObject *str, Py_ssize_t start, \
+.. c:function:: PyObject* PyUnicode_Substring(PyObject *self, Py_ssize_t start, \
```

Comment:
Why not *unicode*?

Reply:
As mentioned below, thought goal was unity with code files. Honestly not sure why I didn't change it anyways, oversight on my part